zebra-consensus/src/checkpoint/tests.rs

```diff
@@ -325,6 +323,11 @@ async fn continuous_blockchain(restart_height: Option<block::Height>) -> Result<
                 break;
             }
 
+            if height == block::Height(0) {
+                // don't try to verify the genesis block
+                continue;
+            }
```

Comment:
If we don't verify the genesis block, then it never gets in the state - so the tests will fail.